src/include/platform/ConfigurationManager.h

```diff
@@ -328,6 +330,13 @@ inline CHIP_ERROR ConfigurationManager::GetPairedAccountId(char * buf, size_t bu
     return static_cast<ImplClass *>(this)->_GetPairedAccountId(buf, bufSize, accountIdLen);
 }
 
+#if CHIP_ENABLE_ROTATING_DEVICE_ID
+inline CHIP_ERROR ConfigurationManager::GetLifetimeCounter(uint16_t & rotationCouter)
```

Comment:
I find the naming a bit inconsistent:

CHIP_ENABLE_ROTATING_DEVICE_ID enables 
`GetLifetimeCounter` which takes a 
`rotationCounter` as argument.

I would at least rename the rotationCounter to lifetimeCounter.

Reply:
renamed